web/post-restoration-transition-application/package.json

```diff
@@ -21,20 +21,19 @@
     "build:test": "pnpm generate && npx serve .output/public"
   },
   "dependencies": {
-    "@sbc-connect/nuxt-core-layer-beta": "^0.0.5",
-    "@vue/compiler-sfc": "^3.5.16",
+    "@sbc-connect/nuxt-business-base": "^0.1.11",
     "@vuepic/vue-datepicker": "^8.8.1",
     "currency-symbol-map": "^5.1.0",
     "http-status-codes": "^2.3.0",
-    "nuxt": "3.16.1"
+    "nuxt": "4.1.2"
   },
   "devDependencies": {
     "@axe-core/playwright": "^4.10.1",
-    "@faker-js/faker": "^9.6.0",
+    "@faker-js/faker": "^10.0.0",
     "@nuxt/eslint": "^1.2.0",
     "@nuxt/image": "^1.10.0",
-    "@nuxt/test-utils": "^3.17.2",
-    "@nuxtjs/i18n": "^9.3.4",
+    "@nuxt/test-utils": "^3.19.2",
+    "@nuxtjs/i18n": "^10.1.0",
```

Comment:
can remove this? I think its better to just pull in whichever version from the layers